config/routes.rb

```diff
@@ -13,4 +13,10 @@
   resources :users, only: [:index, :show] do
     resources :posts, only: [:index, :show] 
   end
+
+  namespace :api, defaults: { format: json } do 
```

Comment:
- [ ] It would be better if we have  API endpoint that returns all comments for a post. For example; `users/:user_id/posts/:id/comments` or `api/users/:user_id/posts/:id/comments`

Reply:
@addod19 ,

Resolved.

![image](https://user-images.githubusercontent.com/10332499/156661908-807957ca-85c8-4172-b338-46e16af56e01.png)